examples/deviceshifu/mockdevice/lwM2M/checkoutput.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+#Default value to write to the mock device
+writeData=88.8
+
+#Get the pod name of deviceshifu
+pod_name=$(kubectl get pods -n deviceshifu -l app=deviceshifu-lwm2m-deployment -o jsonpath='{.items[0].metadata.name}')
+
+if [ -z "$pod_name" ]; then
+    echo "No deviceshifu-lwm2m pod found. Exiting..."
+    exit 1
+fi
+
+# Retrieve LwM2M server information with multiple retries
+for i in {1..30}; do
+    # Check deviceshifu status
+    out=$(kubectl exec -n deviceshifu nginx -- curl deviceshifu-lwm2m.deviceshifu.svc.cluster.local/float_value --connect-timeout 5)
```

Comment:
use can use the provided curl retry mechanism https://everything.curl.dev/usingcurl/downloads/retry.html